package.json

```diff
@@ -27,6 +27,7 @@
     "gray-matter": "^4.0.3",
     "highlight.js": "^11.3.1",
     "highlightjs-solidity": "^2.0.1",
+    "huffc": "Saw-mon-and-Natalie/huffc.git#fileless-parser",
```

Comment:
If anything, why not point to the original repo?

Reply:
The original repo only compiles files and not the content as strings. It also misses declaration files for typescript. 

Once those has been added/modified in the original repo, we can change this to `huffc`.

Reply:
I've created a PR for `huffc` to include the required changes:
[feat: Allow parsing/compiling of content only data ...](https://github.com/huff-language/huffc/pull/40)

Once that PR is reviewed and merged, we can update this `dependency`.

Reply:
That's great thanks :-)